s3tests_boto3/functional/__init__.py

```diff
@@ -484,6 +496,17 @@ def get_tenant_iam_client():
                           use_ssl=config.default_is_secure)
     return client
 
+def get_alt_iam_client():
+
+    client = boto3.client(service_name='iam',
+                          region_name='us-east-1',
```

Comment:
I see region as empty string in few functions.

Reply:
yes it is, could you remove it from this declaration.